test/powershell/Modules/Microsoft.PowerShell.Core/Out-Host.Tests.ps1

```diff
@@ -27,8 +21,9 @@ Describe "Out-Host Tests" -tag CI {
     }
     It "Out-Host writes to host output" {
         $stringToWrite = "thing to write"
+        $stringExpected = "::$($stringToWrite):NewLine"
```

Comment:
It's just less intuitive to construct the expected string if you are using a variable. But currently there is only one instance where `$()` is required. Let's keep it for now, and we can always change to something else if there are any complaints.